loops.js

```diff
@@ -1,4 +1,4 @@
-// ===========================================
```

Comment:
vanaf regel 49 : door if (i % 2 == 0) te gebruiken neem je ook de 0 mee in je brekening.
Er hoeft dan geen extra statement te worden gemaaakt voor 0.

if (i %2 === 0){
        console.log(i+" is even ")
    }
    else {
        console.log(i+" is oneven ")